.env.example

```
@@ -408,3 +408,287 @@ NEUROLINK_DISABLE_MCP_TOOLS=false
+# Development
+DEV_AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
+DEV_AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG
+
+# Staging
+STAGING_AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
+STAGING_AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG
+
+# Production
+PROD_AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
+PROD_AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG
+PROD_AWS_REGION=us-east-1
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The PROD_AWS_REGION key should go before the PROD_AWS_SECRET_ACCESS_KEY key

(UnorderedKey)